Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/generator/BukkitPlotGenerator.java

```diff
@@ -88,8 +88,8 @@ public BukkitPlotGenerator(final String world, final ChunkGenerator cg) {
                 Set<PlotArea> areas = PlotSquared.get().getPlotAreas(name);
                 if (!areas.isEmpty()) {
                     PlotArea area = areas.iterator().next();
-                    if (!area.MOB_SPAWNING) {
-                        if (!area.SPAWN_EGGS) {
+                    if (!area.isMobSpawning()) {
```

Comment:
The naming for these is a bit weird, although standard. I don't really mind it, but it could be a bit confusing.

It kind of applies to most boolean getters, so I don't know if it's worth changing. I would like to hear some opinions on this, however.